default-menu-auto-hide-fedora.ks.in

```diff
@@ -0,0 +1,17 @@
+#version=DEVEL
+#test name: default-menu-auto-hide
+%ksappend repos/default.ks
+
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+%post --nochroot
+SYSROOT=/mnt/sysroot
+
+grep -q "menu_auto_hide=1" $SYSROOT/boot/grub2/grubenv || \
+     echo "*** Failed check: The menu_auto_hide is not part of the /boot/grub2/grubenv by default" >> $SYSROOT/root/RESULT
+
+if [[ ! -e $SYSROOT/root/RESULT ]]; then
```

Comment:
There is a fragment that you can use instead.

Reply:
Fixed.